lib/compress/zstd_compress_internal.h

```diff
@@ -1264,6 +1264,24 @@ MEM_STATIC void ZSTD_debugTable(const U32* table, U32 max)
 
 #endif
 
+/* Short Cache */
+
+typedef enum {
+    /* TODO: this is a good place to document short cache */
+    ZSTD_tfp_forCCtx = 0,
+    ZSTD_tfp_forCDict = 1
+} ZSTD_tableFillPurpose_e;
+
+#define ZSTD_SHORT_CACHE_TAG_BITS 8
+#define ZSTD_SHORT_CACHE_TAG_MASK ((1u << ZSTD_SHORT_CACHE_TAG_BITS) - 1)
+
+/* Helper function for ZSTD_fillHashTable and ZSTD_fillDoubleHashTable */
+MEM_STATIC void writeTaggedIndex(U32* const hashTable, size_t hashAndTag, U32 index) {
```

Comment:
This symbol should still have a `ZSTD_` prefix, even if it is static  / internal.